templates/registration/login.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <h5 style = "color: #a52222">wrong credentials</h5>
+  {% endif %}
+
+  <form method="post" action = {% url "login" %}>
```

Comment:
The `action` attribute in the form tag should be enclosed in quotes. It should be `action="{% url 'login' %}"` to ensure proper URL resolution.